### ISSUE TITLE
Add mail.preview config to allow simpler conditional enabling, open door to other enhancments.

### ISSUE THREAD

**Issue by coderabbi** [themsaid/laravel-mail-preview#28]
Would you consider a pull request to change the logic of `laravel-mail-preview/src/MailProvider.php` to call `$this->registerPreviewSwiftMailer()` if *either* the current condition is met *or* `$this->app['config']['mail.preview'] == true`?
 
The rationale is that by providing a config key which can enable mail previews without replacing the mail driver, (1) it becomes simpler to condition mail previews upon the environment, APP_DEBUG value, or similar (certainly possible as is, but more cumbersome), and (2) you open the door to enhancements akin to that in #7 (currently closed, I realize).

I'll provide the PR, just wanted to confirm it would be well received first. Thanks!

**Comment by themsaid** [comment]
Hello :)

Yes please go ahead and open a PR, it sounds cool to me.

**Comment by coderabbi** [comment]
Awesome, thanks!

One odd bit of adding this and maintaining BC (obviously need to do that here), is that it would be possible to have mail.preview == false and mail.driver == preview in which case just looking at mail.preview would lead you to beleive that they were turned off, but , of course, they would be turned on.
 
I don't think there's any way to escape that, and I think it's still worth it, but just wanted to note that after I sugested this, I realized it would be introducing a potentially confusing bit, albeit minor.
